smart-contracts/contracts/BridgeBank/BridgeBank.sol

```diff
@@ -546,17 +689,17 @@ contract BridgeBank is BankStorage,
             amount,
             lockBurnNonce,
             decimals,
-            symbol,
```

Comment:
Let's remove lines 680 and 681, this will save us on gas as we already have this stuff on the stack when emitting the log event. The reason this will result in a gas savings is due to not having to read from storage to fetch the name and symbol.